### DISCUSSION TITLE
[ideas] Allow publish of specific solution instead of all customization

### DISCUSSION THREAD

**Discussion by sanjeev40084** [microsoft/powerplatform-vscode#574] [Ideas]
Currently the cli only allows publish of all customizations of all solutions from the connected instance. i.e. 
`pac solution publish`

It would nice to be able to just publish specific solution from the instance e.g.
`pac solution publish [solution name]`

looks like this was added in backlog about 2 years ago #93 but since there has not been any update, thought to add here as ideas. 😎

**Comment by petrochuk** [comment]
If there is enough interest it is possible to add call to [PublishXmlRequest](https://learn.microsoft.com/en-us/dotnet/api/microsoft.crm.sdk.messages.publishxmlrequest?view=dataverse-sdk-latest)
